pkg/providers/vault/examples/nginx-pod-keyvault.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: nginx-keyvault
```

Comment:
Should we rename these to `-vault` instead of `-keyvault` to distinguish this example from the Azure Key Vault one.